tests/continuous/test_consistent.py

```diff
@@ -0,0 +1,239 @@
+"""
+Tests for scores.continuous.consistent_impl
```

Comment:
I don't know enough about these scores to review the tests for correctness. I will check them for things like handling edge cases or other obvious issues. The coverage is 100%, which is great.

Reply:
It all looks reasonable to me.